docs/technical_reference/flowsheets/mvc.rst

```diff
@@ -0,0 +1,126 @@
+system. The flowsheet relies on the following key assumptions:
+
+   * supports steady-state only
+   * property package(s) supporting liquid and vapors is provided
+
+.. figure:: ../../_static/flowsheets/mvc.png
+    :width: 600
+    :align: center
+
+    Figure 1. MVC flowsheet
+
+Documentation for each of the WaterTAP unit models can be found below.
+    * `Pressure Changer <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/pressure_changer.html>`_
+    * `Evaporator <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/evaporator.html>`_
+    * `Compressor <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/compressor.html>`_
+    * `Condenser <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/condenser.html>`_
+
+Documentation for each of the IDAES unit models can be found below.
+    * `Feed <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/feed.html>`_
+    * `Heat Exchanger <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/heat_exchanger.html>`_
+    * `Separator <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/separator.html>`_
+    * `Product <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/product.html>`_
+    * `Mixer <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/mixer.html>`_
+    * `Translator <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/translator.html>`_
+
+Documentation for each of the property models can be found below.
+    * `Water <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/water.html>`_
+    * `Seawater <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/seawater.html>`_
+
+Documentation for the costing relationships can be found below.
+    * `WaterTAP Costing Package <https://watertap.readthedocs.io/en/latest/technical_reference/costing/watertap_costing.html>`_
+
+The objective function is to minimize the levelized cost of water, which can be represented by the following equation
+where :math:`Q` represents volumetric flow, :math:`f_{crf}` represents capital recovery factor
+:math:`C_{cap,tot}` represents total capital cost, :math:`C_{op,tot}` represents total operating cost, and
+:math:`f_{util}` represents the utilization factor:
+
+    .. math::
+
+        LCOW_{Q} = \frac{f_{crf}   C_{cap,tot} + C_{op,tot}}{f_{util} Q}
+
+Degrees of Freedom
+------------------
+The following variables are initially specified for simulating the MVC flowsheet (i.e., degrees of freedom = 0):
+    * feed water conditions (mass flow, mass fractions, temperature, and pressure)
+    * feed pump efficiency and pressure change (ΔP)
+    * separator split fraction
```

Comment:
```suggestion
```
I would remove the separator split fraction because it is not fixed for simulating the flowsheet, but an initial guess of the recovery is provided.